multiversx_sdk/governance/resources.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class VoteType(Enum):
+    YES = "796573"  # "yes" hex-encoded
+    NO = "6e6f"  # "no" hex-encoded
+    ABSTAIN = "6162737461696e"  # "abstain" hex-encoded
+    VETO = "7665746f"  # "veto" hex-encoded
```

Comment:
Can we do, perhaps:

```
YES = b"yes".hex()
...
```

Or, actually, even better - simply hold the pretty strings here, but do the hex encoding in the factory? Even the encoding is doubled hex encoding (unfortunate, for now)?